README.md

```diff
@@ -1,54 +1,60 @@
+
 ## Introduction
 
 The Looker JavaScript Embed SDK is designed to facilitate using Looker embedded content in your web application. The goal is to make communication between a host website and one or more embedded dashboards, looks, explores and extensions easier and more reliable.
 
-The Looker JavaScript Embed SDK typically uses embed SSO to sign an embed url in order to authenticate the user of the embed. This mechanism relies on Looker cookies being available to the embedded IFRAME in order for the application to work. Looker also provides a mechanism that allows embedded Looker IFRAMES to work without the need for cookies. Details can be found [here](#cookieless). Embed SDK functionality that will not work with cookieless embed is identified in this document.
+The Looker JavaScript Embed SDK typically uses embed SSO to sign an embed url in order to authenticate the user of the embed. This mechanism relies on Looker cookies being available to the embedded IFRAME in order for the application to work. Looker also provides a mechanism that allows embedded Looker IFRAMES to work without the need for cookies. Details can be found [here](#cookieless).
 
 A typical setup might look like this. In this case, a dashboard with an id of `11` is created inside a DOM element with the id `dashboard`. The `dashboard:run:start` and `dashboard:run:complete` events are used to update the state of the embedding window's UI, and a button with an id of `run` is scripted to send a `dashboard:run` message to the dashboard.
 
 ```javascript
-LookerEmbedSDK.init('looker.example.com', '/auth')
+getEmbedSDK().init('looker.example.com', '/auth')
 
-const setupDashboard = (dashboard) => {
+const setupDashboard = (connection) => {
   document.querySelector('#run').addEventListener('click', () => {
-    dashboard.send('dashboard:run')
+    connection.asDashboardConnection().run()
   })
 }
 
-LookerEmbedSDK.createDashboardWithId(11)
-  .appendTo('#dashboard')
-  .on('dashboard:run:start', () => updateState('#dashboard-state', 'Running'))
-  .on('dashboard:run:complete', () => updateState('#dashboard-state', 'Done'))
-  .build()
-  .connect()
-  .then(setupDashboard)
-  .catch((error: Error) => {
-    console.error('An unexpected error occurred', error)
-  })
+try {
+  connection = await getEmbedSDK()
+    .createDashboardWithId(11)
+    .appendTo('#embed_container')
+    .on('dashboard:run:start', () => updateStatus('Running'))
+    .on('dashboard:run:complete', () => updateStatus('Done'))
+    .build()
+    .connect()
+  setupDashboard(connection)
+} catch (error) {
+  console.error('An unexpected error occurred', error)
+}
 ```
 
-A more complete example can be found [here](demo/demo.ts). Detailed instructions on how to use it are [here](#demo).
+A more complete example can be found [here](demo/demo_single_frame.ts) and [here](demo/demo_multi_frame.ts). Detailed instructions on how to use the SDK can be found [here](#demo).
```

Comment:
these links are not resolving on the npm site

Reply:
I've fixed this to use a full URL. It will be incorrect until this PR is merged